src/Network/Send/kRO/RagexeRE_2015_11_04a.pm

```diff
@@ -15,6 +15,11 @@ package Network::Send::kRO::RagexeRE_2015_11_04a;
 use strict;
 use base qw(Network::Send::kRO::RagexeRE_2015_10_29a);
 
+# TODO: remove 'use Globals' from here, instead pass vars on
+use Globals qw(%config $bytesSent %packetDescriptions $enc_val1 $enc_val2 $char $masterServer $syncSync $accountID %timeout %talk $skillExchangeItem $net $rodexList $rodexWrite %universalCatalog %rpackets);
```

Comment:
i think the only variable you will need is `$char`

Reply:
Hello, you were right that I could remove unused variables. I actually only need $char and $rodexWrite. I did push the updated files to my repo.